alexandria/core/models.py

```diff
@@ -292,9 +301,6 @@ def get_webdav_url(self, username, group, host="localhost:8000"):
         return f"{settings.ALEXANDRIA_MANABI_DAV_SCHEME}://{host}{settings.ALEXANDRIA_MANABI_DAV_URL_PATH}/{token.as_url()}"
 
     def create_thumbnail(self):
-        if not settings.ALEXANDRIA_ENABLE_THUMBNAIL_GENERATION:
-            return
-
```

Comment:
I'd prefer to keep this here. Otherwise, all the call sites of this method will need to check the setting

Reply:
I even think that we should do the full check (`self.variant == File.Variant.ORIGINAL and self.renderings.count() == 0`) in here as well. Make the call site need as little logic as possible

Reply:
Moved into methods